core/build.gradle

```diff
@@ -194,3 +196,43 @@ if (tasks.findByName('trimShadedJar')) {
         dontnote
     }
 }
+
+class PublicSuffixesTask extends DefaultTask {
+    // Rebuild up to once in 7 days, unless cleaned.
+    @Input
+    final def days = (System.currentTimeMillis() / TimeUnit.DAYS.toMillis(7)).intValue()
+    @OutputFile
+    final def publicSuffixesFile = project.file("${project.buildDir}/resources/main/com/linecorp/armeria/public_suffixes.txt")
```

Comment:
How about writing to `src/main/resources/..` so we know when the suffix list changes? Please also add the generated file into this pull request.

Reply:
Yes the file is added. However when I changed OutputFile to `projectDir/src/main/resources/...` it couldn't find that file in the code. I wonder if that is because of this line 🤔?https://github.com/line/armeria/blob/d24db55953391e0101d2f59ede1c4c889ef6e0e9/core/build.gradle#L232

https://ci.appveyor.com/project/line/armeria/builds/36497460/job/d0ji8ok3dnj2e384#L1121